src/js/core.js

```diff
@@ -341,6 +341,7 @@ const photoBooth = (function () {
     };
 
     api.cheese = function (photoStyle, retry = 0) {
+        api.stopPreviewVideo(CameraDisplayMode.COUNTDOWN);
         photoboothTools.console.logDev('Photostyle: ' + photoStyle);
```

Comment:
?

Reply:
As I said I tried a bit around with it and tried to keep the cheese (Wouldn’t make any sense though. The photo was taken at the time of the fade to white) That‘s why I considered the branch dirty. This line can be deleted.